app/controllers/reader/documents_controller.rb

```diff
@@ -56,6 +60,24 @@ def documents
     end
   end
 
+  def capture_document_mismatch(ids)
+    appeal_docs = Document.where(file_number: appeal.veteran_file_number).where.not(id: ids)
+
+    if appeal_docs.present?
+      error = DocumentCountMismatch.new("Document count mismatch")
+      Raven.capture_exception(
+        error,
+        extra: {
+          appeal_id: appeal.external_id,
+          document_service: appeal.document_fetcher.document_service.name,
+          missing_doc_ids: appeal_docs.pluck(:id),
+          message: "`appeal.document_fetcher.find_or_create_documents!.map(&:id)` vs " \
+                   "`Document.where(file_number: appeal.veteran_file_number)` "
+        }
```

Comment:
Suggest adding counts for good measure:
```suggestion
          document_fetcher_doc_count: ids.count,
          documents_for_vet_doc_count: Document.where(file_number: appeal.veteran_file_number).count,
        }
```